src/plugins/normalize-whitespace/demo.html

```diff
@@ -0,0 +1,48 @@
+<body>
```

Comment:
Why HTML? We can use HTML elements in Markdown, wouldn't that work?

Reply:
I just left it in its previous format. But yeah, switching to `demo.md` is no problem. Done!

Reply:
I meant as a part of its README, not a separate `demo.md`. Ideally none of them should have anything other than a `README.md`.

Reply:
Hm, I think there might be a reason for this, but I'm not sure. I'll figure it out.